script.js

```diff
@@ -11,12 +15,25 @@ window.onload = function () {
     let container = document.getElementById('note-container');
     let button = document.getElementById('add-button');
 
-    let json = sessionStorage.getItem('notes');
-    if (json === null) {
+    let notesJson = sessionStorage.getItem('notes');
+    if (notesJson === null) {
+        return;
```

Comment:
looks like we can't read settings if no notes saved

Reply:
Fixed